extension/asset.go

```diff
@@ -47,70 +47,103 @@ func BuildAssetsForExtensions(shopwareRoot string, extensions []Extension) error
 		for _, entry := range cfgs {
 			// If extension has package.json install it
 			if _, err := os.Stat(fmt.Sprintf("%s/Resources/app/administration/package.json", entry.BasePath)); err == nil {
-				npmInstallCmd := exec.Command("npm", "--prefix", fmt.Sprintf("%s/Resources/app/administration/", entry.BasePath), "install") //nolint:gosec
-				npmInstallCmd.Stdout = os.Stdout
-				npmInstallCmd.Stderr = os.Stderr
-				err := npmInstallCmd.Run()
+				err := npmInstall(fmt.Sprintf("%s/Resources/app/administration/", entry.BasePath))
 
 				if err != nil {
 					return err
 				}
 			}
 		}
 
-		npmInstallCmd := exec.Command("npm", "--prefix", fmt.Sprintf("%s/src/Administration/Resources/app/administration/", shopwareRoot), "install") //nolint:gosec
-		npmInstallCmd.Stdout = os.Stdout
-		npmInstallCmd.Stderr = os.Stderr
-		err := npmInstallCmd.Run()
+		if _, err := os.Stat(fmt.Sprintf("%s/vendor/shopware/administration/Resources/app/administration/package.json", shopwareRoot)); err == nil {
```

Comment:
Can we build this similar to https://github.com/shyim/shopware-docker/blob/v1/functions.sh#L183-L193 ?